src/lib/styles/global/utility-classes.scss

```diff
@@ -51,3 +51,9 @@
 .no-margin {
   margin: 0 !important;
 }
+
+// Format numbers as monospaced figures
+
+.num {
```

Comment:
nitpick: Should all the numbers be monospaced? If not, probably more precise name would fit better (`tabular-num`?)

Reply:
Makes sense sure, done